src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -47,12 +47,12 @@ export const MessagingThreadListItem: React.FC<
     children,
     className,
     icon,
-    isRead,
+    isRead = true,
     timestamp,
     title,
     hasDraft,
     selected,
-    status,
+    status = "active",
```

Comment:
The intent with this commit is to make `MessagingNewThread` unnecessary by making `MessagingThreadListItem` flexible enough for both cases. As part of that, I made some props optional and also defaulted them to values that make the display the most "normal". 

In general, the merging into a single component seems to work pretty well. There is one visual bug I noticed where when displaying with a draft icon and only a title and no `children`, the title is slightly misplaced and not aligned with the draft icon, but I didn't want to block this PR on fixing that.

<img width="982" alt="Screen Shot 2020-07-14 at 3 51 38 PM" src="https://user-images.githubusercontent.com/39533986/87484036-0f6c5b00-c5ea-11ea-959a-78ce7e967a08.png">

Reply:
Hm, looks like both the title _and_ draft indicator are misaligned in this scenario, [as compared to the mocks](https://www.figma.com/file/paAte3jkRiLpgN0i9TvPbN/Clever-Messaging?node-id=1276%3A37183). Note: Is the linked mock still the most up-to-date for this?

I believe this breaks the new-thread-with-draft use case, which is currently working on the Teacher and Student pages. I'm a little reluctant to do that, if at all avoidable. 😕

Reply:
@clcaelynn-clever Just pushed a commit that I think fixes this. It ended up kind of clunky, but a simpler way was escaping me (if you see one let me know 😄 )

<img width="982" alt="Screen Shot 2020-07-14 at 5 33 50 PM" src="https://user-images.githubusercontent.com/39533986/87489690-7e50b080-c5f8-11ea-9e41-8973e2483d74.png">

_Smiley Dude once more has reason to smile_

Reply:
A simpler way isn't obvious to me, so I think we can definitely take function over form here! Can be refactored at any time, after all - launching on-time is a priority right now. 😅 

Thanks for resolving this!